alg/teca_lapse_rate.py

```diff
@@ -0,0 +1,168 @@
+import numpy as np
+
+def calculate_lapse_rate(t,z,zs,zmax = 9000):
```

Comment:
please move this into the `teca_lapse_rate` class. as is it will be introduced at module scope, but it's really an implementation detail that need not be exposed and will keep the module cleaner and easier to use to encapsulate this in the class that uses it.